### PR TITLE
Add processing and notification responses

### PR DESCRIPTION
**Pull request by caleeli**

### CODE REVIEW

File: ProcessMaker/Jobs/InstanceAction.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            //Load process instance
+            $instance = $workflow->getEngine()->loadExecutionInstance($this->instanceId);
+
+            //Do the action
+            App::call([$this, 'action'], compact('workflow', 'process', 'instance'));
+
+            //Run engine to the next state
+            $workflow->getEngine()->runToNextState();
+        } catch (\Throwable $t) {
+            dd($t);
```
Comment (velkymx) [inline ProcessMaker/Jobs/InstanceAction.php]: Move to Laravel log

File: ProcessMaker/Jobs/StartEvent.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+            //Do the action
+            $response = App::call([$this, 'action'], compact('workflow', 'process', 'event'));
+
+            //Run engine to the next state
+            $workflow->getEngine()->runToNextState();
+            
+            return $response;
+        } catch (\Throwable $t) {
+            dd($t);
```
Comment (velkymx) [inline ProcessMaker/Jobs/StartEvent.php]: Log

File: ProcessMaker/Jobs/TokenAction.php [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            $activity = $element;
+            $data = $this->data;
+
+            //Do the action
+            App::call([$this, 'action'], compact('workflow', 'instance', 'token', 'element', 'activity', 'data'));
+
+            //Run engine to the next state
+            $workflow->getEngine()->runToNextState();
+        } catch (\Throwable $t) {
+            dd($t);
```
Comment (velkymx) [inline ProcessMaker/Jobs/TokenAction.php]: Log

File: ProcessMaker/Repositories/ExecutionInstanceRepository.php [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+     *
+     * @param string $uid
+     *
+     * @return \ProcessMaker\Nayra\Contracts\Engine\ExecutionInstanceInterface
+     */
+    public function loadExecutionInstanceByUid($uid, StorageInterface $storage)
+    {
+        $instance = Instance::where('uid', $uid)->first();
+        if (!$instance)
+            return;
```
Comment (velkymx) [inline ProcessMaker/Repositories/ExecutionInstanceRepository.php]: Is there a better way to return?

File: resources/views/layouts/layout.blade.php [hunk 5]
```diff
@@ -6,7 +6,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="user-uid" content="{{\Auth::user()->uid}}">
     <meta name="csrf-token" content="{{ csrf_token() }}">
+    @if(Auth::user())
+    <meta name="user-uid" content="{{ Auth::user()->uid }}">
```
Comment (velkymx) [inline resources/views/layouts/layout.blade.php]: Duplicate line

File: resources/views/layouts/layout.blade.php [hunk 6]
```diff
@@ -6,7 +6,15 @@
...
+    <meta name="user-id" content="{{ Auth::user()->id }}">
+    @endif
     <meta name="api-token" content="{{ session('apiToken')['access_token']}}">
+    <meta name="broadcaster" content="{{config('broadcasting.broadcaster')}}">
```
Comment (velkymx) [inline resources/views/layouts/layout.blade.php]: Move broadcaster meta into @if(config('broadcasting.broadcaster') block

**Comment by caleeli** [comment]
- Added Laravel Log to register fails during the flow execution.
- Trigger a 404 exception when a instance does not exists in the DB
- Add validation to load socket.io parameters in the meta headers
- Updated composer.lock to the latest Nayra revision.

**Pull request merged**